src/twisted/protocols/_smb/core.py

```diff
@@ -0,0 +1,760 @@
+# -*- test-case-name: twisted.protocols._smb.tests -*-
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+"""Implement Microsoft's Server Message Block protocol"""
```

Comment:
Codecov is extremely mad about this file, I think because `tests.py` is in the wrong place: it should be in a `test` directory somewhere (`twisted/protocols/_smb/test/` would be fine), and named `test_smb.py` rather than just `tests`.